docs/91_version_history.rst

```diff
@@ -35,229 +35,186 @@ Release Date: 2023-10-17
-       - A number of optimisations to the SQLite in-memory storage plugin and the SQLiteLB storage plugin have been added that increase the rate at which readings can be stored with these plugins.
-       - The documentation of the OMF North plugin has been updated to conform with the latest look and feel of the configuration user interface. It also contains notes regarding the use of complex types verses the OMF 1.2 linked types.
-       - The Simple Rest south plugin documentation has been updated to include the new authentication options. Th plugin configuration has also been updated to make use of the new tab based display, grouping relating configuration options together.
-       - The PI Server South plugin new responds properly if AF Attributes are edited or edited, and if PI Points are deleted. Unfortunately, it is not possible to detect PI Point attribute edits. It is not possible to detect creation of new AF Attributes or PI Points that match the query parameters. If this occurs, you must restart your PI Server South instance.
-       - A new filter plugins has been added that will allow data to be normalised to a particular reading rate within a stream.
-       - A new filter has been added that will allow two assets to be joined with a pipeline.
-       - A new feature has been added that allows pipelines to be defined that may be applied to control operations. This allows the filter pipelines to manipulate the control requests that arrived from various sources within Fledge.
-       - The North service could crash if it retrieved invalid JSON while processing a reconfiguration request. This was addressed by adding an exception handler to prevent the crash.
+       - The documentation for the control notification plugin has been updated to include examples for all destinations of control requests.
        - Windowed averages in the notification service preserve the type of the input data when creating the averages. This does not work well for integer values and has been changed such that integer values are promoted to floating point when using windowed averages for notification rule input.
        - The notification service documentation has been updated to include examples of notifications based on statistics and audit logs.
        - The notification mechanism has been updated to accept raw statistics and statistics rates as an input for notification rules. This allows alerts to be raised for pipeline flows and other internal tasks that generate statistics.
-       - The control dispatcher now has access to the audit logging system.
        - Notifications can now register for audit log entries to be sent to notification rules. This allows notification to be made based on internal state changes of the system.
+       - The north opcuaclient has been updated to support multiple values in a single write.
+       - The north opcuaclient plugin has been updated to support security mode and security policy.
```

Comment:
I would mention OPC UA here, as in "to support OPC UA security modes and security policies." There are more than one of each so I made them plural.